src/libraries/System.Runtime/ref/System.Runtime.cs

```diff
@@ -12677,18 +12677,30 @@ public enum ResourceLocation
         ContainedInAnotherAssembly = 2,
         ContainedInManifestFile = 4,
     }
+    [System.ComponentModel.EditorBrowsable(System.ComponentModel.EditorBrowsableState.Never)]
```

Comment:
```suggestion
    [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
```
ref sources should use the full type name for attributes including Attribute suffix. It is the style used by the ref sources auto-generator.

(multiple places)